src/devtools/views/Components/TreeContext.js

```diff
@@ -780,4 +774,15 @@ function recursivelySearchTree(
   );
 }
 
+function getNearestResult(
+  searchResults: Array<number>,
+  selectedElementID: number | null
+) {
+  const result = searchResults.findIndex(
+    value => value >= ((selectedElementID: any): number)
```

Comment:
Whoops! I overlooked something 😁

We shouldn't be comparing IDs here, but indices. These won't necessarily correlate. I'll make a follow up fix!